vortex-layout/src/layouts/stats/writer.rs

```diff
@@ -74,7 +74,7 @@ impl StatsLayoutWriter {
 impl LayoutWriter for StatsLayoutWriter {
     fn push_chunk(
         &mut self,
-        segments: &mut dyn SegmentWriter,
+        segment_writer: &mut dyn SegmentWriter,
```

Comment:
I don't mind either way on this one. Where the context is so obvious, I lean towards the terse version

Reply:
the issue here is that in some callsites (like `VortexWriteOptions`) we have multiple `segment` things and I personally find it harder to keep track.